README.md

```diff
@@ -0,0 +1,93 @@
+# Go Snowflake Driver
+
+<a href="https://github.com/snowflakedb/gosnowflake/actions?query=workflow%3A%22Build+and+Test%22">
+    <img src="https://github.com/snowflakedb/gosnowflake/workflows/Build%20and%20Test/badge.svg?branch=master">
+</a>
+<a href="http://www.apache.org/licenses/LICENSE-2.0.txt">
+    <img src="http://img.shields.io/:license-Apache%202-brightgreen.svg">
+</a>
+<a href="https://goreportcard.com/report/github.com/snowflakedb/gosnowflake">
+    <img src="https://goreportcard.com/badge/github.com/snowflakedb/gosnowflake">
+</a>
+
+This topic provides instructions for installing, running, and modifying the Go Snowflake Driver. The driver supports Go's [database/sql](https://golang.org/pkg/database/sql/) package.
+
+# Prerequisites
+
+The following software packages are required to use the Go Snowflake Driver.
+
+## Go
+
+The latest driver requires the [Go language](https://golang.org/) 1.17 or higher. The supported operating systems are Linux, Mac OS, and Windows, but you may run the driver on other platforms if the Go language works correctly on those platforms.
+
+
+# Installation
+
+Get Gosnowflake source code, if not installed.
+
+```sh
+go get -u github.com/snowflakedb/gosnowflake
+```
+
+# Docs
+
+For detailed documentation and basic usage examples, please see the documentation at
+[godoc.org](https://godoc.org/github.com/snowflakedb/gosnowflake/).
+
+# Sample Programs
+
+Snowflake provides a set of sample programs to test with. Set the environment variable ``$GOPATH`` to the top directory of your workspace, e.g., ``~/go`` and make certain to 
+include ``$GOPATH/bin`` in the environment variable ``$PATH``. Run the ``make`` command to build all sample programs.
+
+```
+make install
+```
+
+In the following example, the program ``select1.go`` is built and installed in ``$GOPATH/bin`` and can be run from the command line:
+
+```
+SNOWFLAKE_TEST_ACCOUNT=<your_account> \
+SNOWFLAKE_TEST_USER=<your_user> \
+SNOWFLAKE_TEST_PASSWORD=<your_password> \
+select1
+Congrats! You have successfully run SELECT 1 with Snowflake DB!
```

Comment:
You might want to separate what the reader should type (line 49-52) from the output that the reader will see after the command executes (line 53).

Reply:
Not a way to do it markdown, as it does not support formatting within code blocks.

 I inserted a blank line to give a little separation.

Reply:
That's fine for now.
But for future reference, remember that you could split this into two code blocks.